apps/judicial-system/web/src/routes/Shared/Cases/Cases.tsx

```diff
@@ -176,9 +176,13 @@ export const Cases: React.FC = () => {
           getInvestigationCaseCourtSections(caseToOpen, user),
         ).href
       } else {
-        // Route to Indictment Overivew section since it always a valid step and
+        // Route to Indictment Overview section since it always a valid step and
         // would be skipped if we route to the last valid step
-        routeTo = getIndictmentsCourtSections(caseToOpen).children[0].href
+        const routeToOpen =
+          getIndictmentsCourtSections(caseToOpen).children[0].href ||
```

Comment:
Weird that typescript dose not give an error here.
```suggestion
          getIndictmentsCourtSections(caseToOpen).children[0]?.href ||
```